test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -15,6 +15,190 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+
+    describe('createFunction', function(){
+
+      beforeEach(function(){
+        var self = this;
+        return self.queryInterface.dropFunction('create_job',[{type:'varchar',name:'test'}]).catch(function(err){
```

Comment:
dont throw error in `beforeEach` , extract this test as separate `it` spec , Also use a better way to [check for errors](http://chaijs.com/api/bdd/#method_throw)